src/commonMain/kotlin/fr/acinq/eclair/channel/Helpers.kt

```diff
@@ -363,6 +363,46 @@ object Helpers {
 
         fun isValidFinalScriptPubkey(scriptPubKey: ByteVector): Boolean = isValidFinalScriptPubkey(scriptPubKey.toByteArray())
 
+        fun btcAddressFromScriptPubKey(scriptPubKey: ByteVector, chainHash: ByteVector32): String? {
```

Comment:
nit: I would just call this `btcAddressFromScriptPubKey`
We need unit tests for this, I'll create some from Bitcoin Core's reference tests. 
This conversion should also be provided by `bitcoin-kmp`, I'll add it there.

Reply:
You can use test data from https://github.com/ACINQ/bitcoin-kmp/blob/3b64f865daf3abc34208ce428600f8d7256ae851/src/commonTest/kotlin/fr/acinq/bitcoin/BitcoinTestsCommon.kt

Reply:
This function was ported from phoenix-kmm. I forgot to port the unit tests, but they're [here](https://github.com/ACINQ/phoenix-kmm/blob/fix/issue-123/phoenix-shared/src/commonTest/kotlin/fr/acinq/phoenix/app/UtilitiesTest.kt).

Reply:
Added to bitcoin-kmp: https://github.com/ACINQ/bitcoin-kmp/pull/17

Reply:
I believe we need to merge PR #231 so we can get access to the new function in bitcoin-kmp ?

Reply:
Until we're ready to use the latest bitcoin-kmp from this project, I've updated the existing function, and copied the unit tests.

Done in cdeb6db